libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsFilePlayer.kt

```diff
@@ -125,13 +136,24 @@ internal class VoiceInstructionsFilePlayer(
         mediaPlayer?.setVolume(level, level)
     }
 
+    /**
+     * Successful invoke of this method does not change the state.
+     * In order for the target audio stream type to become effective,
+     * this method must be called before prepare() or prepareAsync().
+     */
+    private fun setAudioStreamType() {
+        mediaPlayer?.setAudioStreamType(streamType)
```

Comment:
> * @deprecated use {@link #setAudioAttributes(AudioAttributes)}

```java
* @deprecated use {@link #setAudioAttributes(AudioAttributes)}
```

Reply:
AudioAttributes has different codes for streams and them doesn't match with ours. In GM we can't use it.